src/fosslight_util/compare_yaml.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022 LG Electronics Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+from fosslight_util.constant import LOGGER_NAME
+from fosslight_util.parsing_yaml import parsing_yml
+
+logger = logging.getLogger(LOGGER_NAME)
+VERSION = 'version'
+LICENSE = 'license'
+NAME = 'name'
+
+
+def compare_yaml(before_file, after_file):
+    before_oss_items, _ = parsing_yml(before_file, os.path.dirname(before_file))
+    after_oss_items, _ = parsing_yml(after_file, os.path.dirname(after_file))
+
+    before_items = get_merged_item(before_oss_items)
+    after_items = get_merged_item(after_oss_items)
+
+    new_before = []
+    for bi in before_items:
+        if bi in after_items:
+            after_items.remove(bi)
+        else:
+            new_before.append(bi)
+
+    ADD = "add"
+    DELETE = "delete"
+    CHANGE = "change"
+    PREV = "prev"
+    NOW = "now"
+
+    comp_out = {ADD: [], DELETE: [], CHANGE: []}
+
+    b_name = list(set([bi[NAME] for bi in new_before]))
+    a_name = list(set([ai[NAME] for ai in after_items]))
+
+    removed_name = [bn for bn in b_name if bn not in a_name]
```

Comment:
[Minor] There seems to be no need to convert it to a list.
ex.
```
    b_name = set([bi[NAME] for bi in new_before])
    a_name = set([ai[NAME] for ai in after_items])
    removed_name = b_name - a_name
    changed_name = b_name ^ removed_name
    added_name = a_name ^ changed_name
```

Reply:
1 more opinion.
Can't the following be changed to the difference set?
NOW: 
```    
changed_name = b_name ^ removed_name
added_name = a_name ^ changed_name
```
TO-BE:
```    
changed_name = b_name - removed_name
added_name = a_name - changed_name
```